app/src/main/res/values/colors.xml

```diff
@@ -5,6 +5,6 @@
     <color name="colorAccent">#D81B60</color>
     <color name="white">#ffffff</color>
     <color name="black">#000000</color>
-    <color name="about_us_background">#D0FCF9</color>
+    <color name="about_us_background">#ffffff</color>
```

Comment:
please rename the color to more appropriate name like "fragment_background_color"